apps/src/redux/isRunning.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * This duck module tracks whether the current app is running or not.
+ */
+var TOGGLE_RUNNING = 'isRunning/TOGGLE_RUNNING';
+
+module.exports.default = function (state, action) {
+  var nextState = state || false;
+
+  if (action.type === TOGGLE_RUNNING) {
+    nextState = action.isRunning;
+  }
+  return nextState;
+};
+
+module.exports.toggleRunning = function (isRunning) {
+  return {
+    type: TOGGLE_RUNNING,
+    isRunning: isRunning
+  };
+};
```

Comment:
I suspect you'll want to eventually, assuming more complex actions also want to set the `isRunning` state.

It's always felt weird to me that we'd treat run/reset as a toggle, outside of the UI state.  The state change run->reset is very different from the state change reset->run.